docs/http-authentication.rst

```diff
@@ -3,7 +3,8 @@ Password-Protected Feeds
 
 :program:`Universal Feed Parser` supports downloading and parsing
 password-protected feeds that are protected by :abbr:`HTTP (Hypertext Transfer Protocol)`
-authentication.  Both basic and digest authentication are supported.
+basic authentication. For any other types of authentication, you can handle the
+authentication yourself and then parse the retrieved feed.
 
 
 Downloading a feed protected by basic authentication (the easy way)
```

Comment:
Basic auth still seems to work fine, as requests supports these type of URLs - for anything else it seems like you should handle authentication yourself